src/services/getPreparedTodos.ts

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+
+export function getPrepapredTodos(
```

Comment:
There is a typo in the function name 'getPrepapredTodos'. It should be 'getPreparedTodos'. This will cause issues when trying to call this function elsewhere in the code.